atomic_reactor/constants.py

```diff
@@ -21,9 +21,14 @@
 CONTAINER_RESULTS_JSON_PATH = os.path.join(CONTAINER_SHARE_PATH, RESULTS_JSON)
 CONTAINER_DOCKERFILE_PATH = os.path.join(CONTAINER_SHARE_PATH, DOCKERFILE_FILENAME)
 
+CONTAINER_BUILDAH_BUILD_METHOD = 'buildah'
```

Comment:
Let's change this to `'buildah_bud'` to reflect the fact that only the 'buildah bud' functionality is available through this plugin.